### PR TITLE
port: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/osx/port.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# port
+
+> Gestor de paquetes para macOS.
+> Más información: <https://www.macports.org>.
+
+- Busca un paquete:
+
+`port search {{search_term}}`
```
Comment (navarroaxel) [inline pages.es/osx/port.md:8]:
```suggestion
`port search {{termino_de_busqueda}}`
```

File: pages.es/osx/port.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Instala un paquete:
+
+`sudo port install {{package_name}}`
```
Comment (navarroaxel) [inline pages.es/osx/port.md:12]: Please translate the placeholders for get this easier for non-English speakers.
```suggestion
`sudo port install {{nombre_de_paquete}}`
```

**Review by kant — COMMENTED** [review]
`sudo port install {{nombre_de_paquete}}` is perfect

File: pages.es/osx/port.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Busca un paquete:
+
+`port search {{termino_de_busqueda}}`
+
+- Instala un paquete:
+
+`sudo port install {{nombre_de_paquete}}`
+
+- Lista de paquetes instalados:
```
Comment (navarroaxel) [inline pages.es/osx/port.md:14]: `List` should be used as verb
```suggestion
- Lista los paquetes instalados:
```

File: pages.es/osx/port.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`port installed`
+
+- Actualiza el puerto y obtiene la última lista de paquetes disponibles:
```
Comment (navarroaxel) [inline pages.es/osx/port.md:18]: I think we shouldn't translate the program name
```suggestion
- Actualiza port y trae la última lista de paquetes disponibles:
```

File: pages.es/osx/port.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo port selfupdate`
+
+- Actualiza los paquetes desactualizados:
+
+`sudo port upgrade outdated`
+
+- Elimina versiones antiguas de paquetes instalados:
```
Comment (navarroaxel) [inline pages.es/osx/port.md:26]:
```suggestion
- Remueve versiones antiguas de paquetes instalados:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**